js/data/bucket/symbol_bucket.js

```diff
@@ -14,6 +14,8 @@ var clipLine = require('../../symbol/clip_line');
 var util = require('../../util/util');
 var loadGeometry = require('../load_geometry');
 var CollisionFeature = require('../../symbol/collision_feature');
+var poleOfInaccessibility = require('../../util/polygon_poi');
```

Comment:
An (admittedly unwritten) convention in GL JS is that file names match the names of the classes / functions they export. Can we rename `polygon_poi` -> `find_pole_of_inaccessibility` or similar?